### PR TITLE
Play 2.8 support

### PR DESCRIPTION
**Pull request by lolgab**
Rebase and fix of #1171

### CODE REVIEW

**Review by lefou — COMMENTED** [review]
Look good to me so far. Thank you!

Are there any tests for Play 2.8? I don't remember if we use the same for all play versions or if we need specific ones for each version, in which case I miss any in this PR.

**Comment by lolgab** [comment]
Ok let me check if we have tests for all the versions!

File: contrib/playlib/test/src/mill/playlib/PlayTestsCommon.scala [hunk 1]
```diff
@@ -0,0 +1,25 @@
+package mill.playlib
+
+import mill.util.{TestEvaluator, TestUtil}
+import utest.framework.TestPath
+
+trait PlayTestSuite {
```
Comment (lefou) [inline contrib/playlib/test/src/mill/playlib/PlayTestsCommon.scala:6]: I'd like to have filename and content in sync. Please rename the file to `PlayTestSuite.scala`.

**Review by lefou — COMMENTED** [review]
Thanks for overhauling the play module test suite. Looks reasonable. One file name change would be nice, but otherwise: Great job!

File: contrib/playlib/test/src/mill/playlib/PlayTestsCommon.scala (hunk 1 above)
Comment (lolgab) [reply contrib/playlib/test/src/mill/playlib/PlayTestsCommon.scala:6]: Ops, this was originally `object PlayTestsCommon` 😄
I'm renaming it!

**Comment by lolgab** [comment]
Renamed the file and updated the syntax of one test that I forgot to update before.

**Comment by lolgab** [comment]
Can this be merged now @lefou?

**Review by lefou — APPROVED** [review]
Looks good to me. Thank you!

**Pull request merged**